src/EnergyPlus/DataAirflowNetwork.hh

```diff
@@ -1092,6 +1100,57 @@ namespace DataAirflowNetwork {
 
 	};
 
+	struct PressureControllerProp
+	{
+		// Members
+		std::string Name; // Provide a unique object name
+		std::string ZoneName; // Name of the zone that is being controlled
+		int ZoneNum; // Zone number
+		int AFNNodeNum; // AFN node number
+		std::string ControlObjectType; // The control type to be used for pressure control
+		std::string ControlObjectName; // Corresponding control type name
+		int ControlTypeSet; // Control type set to be used for pressure control
+		int AvailSchedPtr; // Availability schedule pointer
+		int PresSetpointSchedPtr; // Pressure setpoint schedule pointer
+
+		// Default Constructor
+		PressureControllerProp( ) :
+			ZoneNum( 0 ),
+			AFNNodeNum( 0 ),
+			ControlTypeSet( 0 ),
+			AvailSchedPtr( 0 ),
+			PresSetpointSchedPtr( 0 )
+		{}
+
+	};
+
+	struct DisSysCompAirflowProp // OA fan component
+	{
+		// Members
+		std::string Name; // Name of exhaust fan component
+		int SchedPtr; // Schedule pointer
+		Real64 FlowCoef; // Air Mass Flow Coefficient
+		Real64 FlowExpo; // Air Mass Flow exponent
+		Real64 StandardT; // Standard temperature for crack data
```

Comment:
@mjwitte Here are units

	Real64 FlowCoef; // Air Mass Flow Coefficient [kg/s at 1 Pa]
        Real64 FlowExpo; // Air Mass Flow exponent [dimensionless]
	Real64 StandardT; // Standard temperature for crack data [C]
       Real64 StandardP; // Standard borometric pressure for crack data [Pa]
       Real64 StandardW; // Standard humidity ratio for crack data [kg/kg]